src/applications/vaos/new-appointment/components/NoAddressNote.js

```diff
@@ -4,11 +4,11 @@ import NewTabAnchor from '../../components/NewTabAnchor';
 export default function NoAddressNote() {
   return (
     <p>
-      <strong>Note:</strong> To show locations near your home, you need to add
-      your home address to your VA profile.
+      <strong>Note:</strong> To sort your facilities by closest to your home,
```

Comment:
Hmm, doesn't this impact other pages as well? I see that it's also used by `ProviderSortVariant` and transitively `ProviderList` so hard coding this to "facilities" seem incorrect.